HDSAviz/Interactive_vis_toolkit.ipynb

```diff
@@ -0,0 +1,443 @@
+    "                        color=stat_color[c])\n",
+    "    # radial axes\n",
+    "    p.annular_wedge(0, 0, inner_radius-10, outer_radius+10,\n",
+    "                    -big_angle+angles, -big_angle+angles, color=\"black\")\n",
+    "\n",
+    "    # Placement of parameter labels\n",
+    "    xr = radii[0]*np.cos(np.array(-big_angle/2 + angles))\n",
+    "    yr = radii[0]*np.sin(np.array(-big_angle/2 + angles))\n",
+    "\n",
+    "    label_angle = np.array(-big_angle/2+angles)\n",
+    "    label_angle[label_angle < -np.pi/2] += np.pi\n",
+    "    # Placing Labels and Legend\n",
+    "    p.text(xr, yr, df.Parameter, angle=label_angle,\n",
+    "           text_font_size=\"9pt\", text_align=\"center\", text_baseline=\"middle\")\n",
+    "\n",
+    "    p.rect([-40, -40], [18, 0], width=30, height=13,\n",
+    "           color=list(stat_color.values()))\n",
+    "    p.text([-15, -15], [18, 0], text=Cols,\n",
+    "           text_font_size=\"9pt\", text_align=\"left\", text_baseline=\"middle\")\n",
+    "\n",
+    "    \n",
+    "    return p\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Code Below is the interactive plots version"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"\n",
+    "The current program is meant to draw interactive visualizations using BOKEH\n",
+    "and ipywidgets libraries for sensitivity analysis\n",
+    "\n",
+    "Dependencies : \n",
+    "Currently this code needs to be run Inside Ipython notebook.\n",
+    "Python version :2.7\n",
+    "\n",
+    "Note:\n",
+    "Please ensure Basic_Plot.py exists in same directory and necessary data files are\n",
+    "located in c:/HDSAviz_data folder\n",
+    "\n",
+    "Basically files should be organized as : \n",
+    "\n",
+    "C: HDSAviz_data/\n",
+    "C: HDSAviz_data/analysis_CO.txt \n",
+    "C: HDSAviz_data/analysis_CO2.txt \n",
+    "etc.\n",
+    "\n",
+    "C: HDSAviz_data/interactivevislib.py \n",
+    "C: HDSAviz_data/Interactive_vis_toolkit.ipynb\n",
+    "\n",
+    "\"\"\"\n",
+    "\n",
+    "#import Basic_Plot  once Basic_Plot.py file is ammended.\n",
+    "\n",
+    "import os\n",
+    "from bokeh.models.widgets import Panel, Tabs\n",
+    "from bokeh.io import output_file, output_notebook, show, vform\n",
+    "from ipywidgets import FloatSlider\n",
+    "from IPython.html.widgets import interact\n",
+    "from collections import OrderedDict\n",
+    "from bokeh.plotting import figure, show, output_notebook, output_file\n",
+    "\n",
+    "# initializing the range of values for total order (ST) and first order \n",
+    "# (S1) sensitivity tests \n",
+    "ST_Range = 1\n",
+    "S1_Range = 1\n",
+    "\n",
+    "def change_directory():\n",
```

Comment:
Why would you need to change the working directory for this project?  Can't any calls to this function be addressed by using relative paths since we have a defined directory structure for the project?